### PR TITLE
SAK-48372 Syllabus / Messages: cannot add a URL attachment OR select from Resources

### PR DESCRIPTION
**Pull request by kunaljaykam**
https://sakaiproject.atlassian.net/browse/SAK-48372

### CODE REVIEW

File: content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_attach.vm [hunk 1]
```diff
@@ -100,11 +100,12 @@ includeLatestJQuery('sakai_filepicker_attach.vm');
 	</script>
 
 	<script>
-		document.addEventListener("DOMContentLoaded", function(event){ 
-			$('#resourceTabs a').click(function (e) {
+		document.addEventListener('DOMContentLoaded', () => {
+  			const tabLinks = document.querySelectorAll('#resourceTabs .nav-link');
+			tabLinks.forEach(link => link.addEventListener('click', e => {
```
Comment (adrianfish) [inline content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_attach.vm:105]: You don't need the tabLinks variable here. Just append .forEach on to the selector.

Comment (kunaljaykam) [reply content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_attach.vm:105]: It's amazing, how you think :)

Comment (kunaljaykam) [reply content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_attach.vm]: Thanks for the no-variable solution.

File: content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_attach.vm [hunk 2]
```diff
@@ -100,10 +100,12 @@ includeLatestJQuery('sakai_filepicker_attach.vm');
...
-		document.addEventListener("DOMContentLoaded", function(event){ 
-			$('#resourceTabs a').click(function (e) {
-				e.preventDefault();
-				$(this).tab('show');
+		document.addEventListener('DOMContentLoaded', () => {
+			document.querySelectorAll('#resourceTabs .nav-link').forEach(link => {
+				link.addEventListener('click', e => {
+					e.preventDefault();
+					new bootstrap.Tab(link).show();
+				});
```
Comment (adrianfish) [inline content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_attach.vm:108]: I have a question about all of this. Why is bootstrap not doing this for us? You are effectively doing what bootstrap's js should be doing. What am I missing?

File: content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_attach.vm [hunk 3]
```diff
@@ -152,31 +143,31 @@ includeLatestJQuery('sakai_filepicker_attach.vm');
...
 			#if($!googledriveOn)
 			<li class="nav-item #if($!navigating_googledrive) active #end ">
-				<a class="nav-link" href="#pickerGoogleDrive"><span class="fa fa-google" aria-hidden="true"></span> $tlang.getString("googledrive.integration")</a>
+				<button type="button" class="nav-link" data-bs-target="#pickerGoogleDrive" data-bs-toggle="tab" aria-selected="false"><span class="fa fa-google" aria-hidden="true"></span> $tlang.getString("googledrive.integration")</button>
 			</li>
 			#end
 		</ul>
 
 		<div class="container_filepicker">
 			<div class="tab-content panel panel-default joinLabel">
```
Comment (adrianfish) [inline content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_attach.vm:167]: I think panel should be taken out. It's not in bs any more.

**Review by adrianfish — CHANGES_REQUESTED** [review]
I think the panel stuff should be removed. It's not in BS since 4 I think.

**Review by adrianfish — APPROVED** [review]
Cheers Kunal

**Pull request merged**